Task5.3.2

```diff
@@ -0,0 +1,24 @@
+def check_perfect(n):
+    """Takes number and checks if the number is perfect.
+        Returns boolean.
+    """
+    divisors_sum = sum([divisor for divisor in range(1, n) if n % divisor == 0])
+    return divisors_sum == n
+
+
+def generate_numbers():
+    """Infinitely generator to get the numbers.
```

Comment:
Im sorry, but again please describe it more - what numbers do we get here?
Its important to be more informative - there are several functions here and I need to be able to understand what each one of them does based on the documentation.